test/integration/features/cmd-hostfolder.feature

```diff
@@ -0,0 +1,58 @@
+@cmd-hostfolder @command
+
+Feature: hostfolder commands
+  As a user I can perform basic operations of Minishift with hostfolder mount
+
+  Scenario: User should not be able to list hostfolder when it does not exist
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift hostfolder list" succeeds
+     Then stderr should contain
+     """
+     no host folders defined
+     """
+  Scenario: User should be able to add a hostfolder of type SSHFS
+    When executing "minishift start" succeeds
+     Then Minishift should have state "Running"
+    When executing "minishift hostfolder add -t sshfs --source /home/minishift_ci --target /mnt/sda1/myshare myshare" succeeds
+    And executing "minishift hostfolder list" succeeds
+     Then stdout should should match
+     """
+     "myshare\s* sshfs\s* /home/minishift_ci\s* /mnt/sda1/myshare\s* N"
```

Comment:
Ditto. This depends on the platform we are on; even when variables are used, the path will conflict.